### DISCUSSION TITLE
Markup/Markdown nelle opzioni di chechbox e radio button

### DISCUSSION THREAD

**Discussion by maelaborazioni** [globaleaks/globaleaks-whistleblowing-software#4001] [Q&A] [answered]
Buongiorno, volevo chiedere se c'è un modo per, ad esempio, inserire link e applicare il grassetto al testo delle opzioni delle domanda di tipo checkbox (e/o radio button).
Grazie

**Comment by evilaliv3** [comment, marked as answer]
No al momento non l'abbiamo valutata come una opzione in quanto complicherebbe gli export delle risposte al questionario.

Può però sfruttare due alternative:
- le opzioni radio permettono di configurare dei testi che vengono proposti in caso l'utente passi sull'opzione e nel caso l'utente completi la selezione dell'opzione.
- il markdown è supportato nelle descrizioni delle domande
